staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/schemas.go

```diff
@@ -15,9 +15,24 @@
 package model
 
 import (
+	"time"
+
+	"github.com/google/cel-go/checker/decls"
+	"github.com/google/cel-go/common/types"
 	"k8s.io/apiextensions-apiserver/pkg/apiserver/schema"
 )
 
+const (
+	maxRequestSizeBytes = 3000000
+	// chosen as the numbers of digits of the largest 64-bit integer
+	// we add 4 to account for a unit (like ms) plus the quotation marks that will be used
+	maxDurationSizeJSON = 23
```

Comment:
Ooh, that's better than I expected. I was thinking the json format would be like "1h30m5s"

Reply:
Duration/date/time all share the same format in OpenAPI, so I think `maxDurationSizeJSON` was slightly incorrect before - now it's the same as `maxDatetimeSizeJSON`. I've updated the comment and will do some further testing later on.